static/chat-widget/chat-loader.js

```
@@ -1,86 +1,416 @@
+        var widget_width = data.widget_width
+        var widget_height = data.widget_height
+        var widget_position = data.widget_position
+        var show_popup_notification = data.show_popup_notification
+        var delay_showing_popup_notification = data.delay_showing_popup_notification
+  
+        localStorage.setItem("chatbot_image", chatbot_image);
+  
+        if (widget_width === null || widget_width === undefined || widget_width === '') {
+          widget_width = '25%'
+        } else {
+          widget_width = widget_width;
```

Biome (1.9.4)

[error] widget_width is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)